tests/swe_version.phpt

```diff
@@ -2,7 +2,7 @@
 Basic test
 --SKIPIF--
 <?php
-if (!extension_loaded(‘sweph’)) {
+if (!extension_loaded(‘swephp’)) {
```

Comment:
My search & replace failed here because of the difference in single-quotes:

```
> printf %d\\n \'\‘
> -30
```

vs.

```
> printf %d\\n \'\'
> 39
```

Curious.